content/cloud/noops-sql-db.md

```diff
@@ -144,5 +144,5 @@ Database "inspirational-pig" deleted
 
 ## Next Steps
 
-* For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is where you can provide feedback and ask for assistance from the Fermyon Cloud team on all things NoOps SQL Database. 
-* Visit [FAQ](/cloud/faq) for frequently asked questions
+* For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is the place where you can provide feedback and request assistance from the Fermyon Cloud team regarding all matters related to using SQLite Databases in Spin and Fermyon Cloud.
```

Comment:
How do you feel about all the extra words here?  To me "all things SQLite Database" feels more concise than "all matters related to using SQLite Databases in Spin and Fermyon Cloud."  Is the idea that non-English speakers might find the former too idiomatic?